android/android_guide.md

```diff
@@ -31,5 +31,12 @@ If you like staying up-to-date on the latest Android topics, members of our team
 Comfortable with the basics? Here are a few of our favorite resources to get started on intermediate topics.
 - [Android style tips](http://blog.danlew.net/2014/11/19/styles-on-android/) (circa 2014)
 
+### Licenses ###
+When adding a new license or updating an existing one, you need a list of libraries you're using so you can include their licenses. Most dependencies are included in the `build.gradla`e file, under `<dependencies>` but to get a complete list, you can run:
+
+`./gradlew app:dependencies > dependency-report.txt`
```

Comment:
`app:dependenices` will only check the app module and some projects can be multi-module. Can we just do `./gradlew dependencies`?

Reply:
The global command doesn't print the deps because they're assigned to the module: we should tell ppl to run it on all their modules (maybe there's syntax for that?)